packages/react-table/build-single-packages.json

```diff
@@ -0,0 +1,21 @@
+{
+    "packageName": "@patternfly/react-table",
+    "modules": [ 
+        {
+            "module": "components",
+            "moduleType": "esm"
+        },
+        {
+            "module": "deprecated",
+            "moduleType": "esm"
+        },
+        {
+            "module": "dist/js",
+            "moduleType": "cjs"
+        }
+    ],
+    "blackList": [
```

Comment:
Can we instead call this "exclude", since this is a commonly used term with tsconfig's at least which I think is used in the same way as here.

Reply:
done